pesacheck_meedan_bridge/py/main.py

```diff
@@ -0,0 +1,125 @@
+import json
+import re
+
+import requests
+import sentry_sdk
+import settings
+from check_api import create_mutation_query
+from database import PesacheckDatabase, PesacheckFeed
+
+
+def remove_html_tags(input_string):
+    return re.sub(r"<[^>]*>", "", input_string)
+
+
+def fetch_from_pesacheck():
+    url = "https://api.rss2json.com/v1/api.json"
+    feed_url = settings.PESACHECK_URL
+    params = {
+        "rss_url": feed_url,
+        "api_key": settings.PESACHECK_RSS2JSON_API_KEY,
+        "count": 10,
+        "order_by": "pubDate",
+    }
+    response = requests.get(url, params=params, timeout=60)
+    if response.status_code == 200:
+        return response.json().get("items") or []
+    raise Exception("An Error Occurred fetching data from pesacheck")
+
+
+def post_to_check(query):
+    headers = {
+        "Content-Type": "application/json",
+        "X-Check-Token": settings.PESACHECK_CHECK_TOKEN,
+        "X-Check-Team": settings.PESACHECK_CHECK_WORKSPACE_SLUG,
+    }
+    body = dict(query=query)
+    url = settings.PESACHECK_CHECK_URL
+    response = requests.post(url, headers=headers, json=body, timeout=60)
+    if response.status_code == 200:
+        return response.json()
+    raise Exception(response.text)
+
+
+def store_in_database(feed):
+    db = PesacheckDatabase()
+    db.insert_pesacheck_feed(feed)
+    return feed
+
+
+def post_to_check_and_update(feed):
+    categories = json.loads(feed.categories)
+    language = "fr" if "french" in categories else "en"
```

Comment:
What is the structure of `categories`?

Reply:
@kilemensi categories is a list of strings
```"categories": [
"ethiopia",
"short-form",
"english"
]```

To save in database I have stringified it in `Feed` object

Reply:
Not sure I fully follow. Based on this article, https://pesacheck.org/ghushi-taarifa-hii-inayokemea-malalamishi-ya-watanzania-kuhusu-umeme-haijachapishwa-na-tanesco-e21763736856, what would categories be?

Reply:
Kenya, Tanzania, Kiswahili, Short Form

Reply:
Cool...

1. Is it case sensitive?
2. Does check support any other languages besides EN and FR?

Reply:
@kilemensi The API supports ISO-361 code of a language supported by the workspace. 

The workspace given had been set to only English and French but other languages can be added through Workspace Settings.

Reply:
Does it mean if I send say Amharic (AM) through the API, the post will fail @koechkevin ?

Reply:
@kilemensi a valid ISO-361 code not in the workspace will throw an error
```
{"errors":[{"message":"Language Sorry, language value not supported","code":4,"data":{"language":["Sorry, language value not supported"]}}]}
```

Reply:
Cool.. so language MUST BE en or fr @koechkevin ? Or is it possible for any other code?

Reply:
@kilemensi updated this as per https://code4africa.slack.com/archives/C05UW09KPBL/p1708419691250489?thread_ts=1706689163.300799&cid=C05UW09KPBL